samples/nodejs-typescript-food-catalog/infra/entra/entra.manifest.json

```diff
@@ -1,9 +1,9 @@
 {
-  "id": "${{AAD_APP_OBJECT_ID}}",
-  "appId": "${{AAD_APP_CLIENT_ID}}",
+  "id": "${{ENTRA_APP_OBJECT_ID}}",
+  "appId": "${{ENTRA_APP_CLIENT_ID}}",
   "name": "${{APP_NAME}}-${{TEAMSFX_ENV}}",
   "accessTokenAcceptedVersion": 2,
-  "signInAudience": "AzureADMultipleOrgs",
+  "signInAudience": "AzureADMyOrg",
```

Comment:
Since we're not using anything that's specifically single-tenant, shall we keep this to multitenant to support deploying the sample in situations where the Azure subscription is tied to a different directory than the M365 tenant?

Reply:
Updated 👍